### PR TITLE
[RFR] clear facet when changing page

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: src/app/js/public/facet/sagas.js [hunk 1]
```diff
@@ -28,9 +30,18 @@ export function* handleLoadFacetValuesRequest({ payload: { name } }) {
     return yield put(loadFacetValuesSuccess({ name, values }));
 }
 
+export function* clearFacetSaga({ payload: { action } }) {
+    if (action === 'POP') {
```
Comment (fzaninotto) [inline src/app/js/public/facet/sagas.js:34]: please use a constant instead of a string here

File: src/app/js/public/graph/sagas.js [hunk 2]
```diff
@@ -83,7 +83,6 @@ export default function*() {
     yield throttle(
         500,
         [
-            LOAD_CHART_DATA,
```
Comment (fzaninotto) [inline src/app/js/public/graph/sagas.js:86]: rogue change?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: src/app/js/public/graph/sagas.js (hunk 2 above)
Comment (ThieryMichel) [reply src/app/js/public/graph/sagas.js:86]: This action was triggering the saga twice (here in throttle and later in a takeEvery)

File: src/app/js/public/facet/sagas.js (hunk 1 above)
Comment (ThieryMichel) [reply src/app/js/public/facet/sagas.js:34]: No constant exist for this value (it come from react-router-redux)

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**